### PR TITLE
Enable awsecscontainermetricsreceiver- update components and go.mod

### PR DESCRIPTION
**Pull request by hossain-rayhan**
**Description:** <Describe what has changed.>
This change will enable the `awsecscontainermetricsreceiver` in OT components. 

**Link to tracking Issue:**
#457 

**Testing:**
Unit test added.

**Documentation:** 
README added.

### CODE REVIEW

**Comment by hossain-rayhan** [comment]
Hi @tigrannajaryan and @bogdandrutu can I get this reviewed and merged. It's just two line change for enabling the receiver. Thanks in advanced.

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/go-lintpack/lintpack v0.5.2 // indirect
 	github.com/golangci/golangci-lint v1.31.0
-	github.com/google/addlicense v0.0.0-20200622132530-df58acafd6d5
+	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76
```
Comment (tigrannajaryan) [inline go.mod:9]: Why this change?

Comment (hossain-rayhan) [reply go.mod]: Updated to the old version.

**Review by tigrannajaryan — APPROVED** [review]
